src/bindings/python/PyLogCameraTransform.cpp

```diff
@@ -0,0 +1,147 @@
+    DEFAULT->getLinSideBreakValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LIN_SIDE_BREAK.data()));
+
+    std::array<double, 3> DEFAULT_LINEAR_SLOPE;
+    DEFAULT->getLinearSlopeValue(*reinterpret_cast<double(*)[3]>(DEFAULT_LINEAR_SLOPE.data()));
+
+    py::class_<LogCameraTransform, 
+               LogCameraTransformRcPtr /* holder */, 
+               Transform /* base */>(m, "LogCameraTransform")
+        .def(py::init(&LogCameraTransform::Create))
+        .def(py::init([](const std::array<double, 3> & logSideSlope,
+                         const std::array<double, 3> & logSideOffset,
+                         const std::array<double, 3> & linSideSlope,
+                         const std::array<double, 3> & linSideOffset,
+                         const std::array<double, 3> & linSideBreak,
+                         const std::array<double, 3> & linearSlope,
+                         TransformDirection dir) 
+            {
+                LogCameraTransformRcPtr p = LogCameraTransform::Create();
+                p->setLogSideSlopeValue(*reinterpret_cast<const double(*)[3]>(logSideSlope.data()));
+                p->setLogSideOffsetValue(*reinterpret_cast<const double(*)[3]>(logSideOffset.data()));
+                p->setLinSideSlopeValue(*reinterpret_cast<const double(*)[3]>(linSideSlope.data()));
+                p->setLinSideOffsetValue(*reinterpret_cast<const double(*)[3]>(linSideOffset.data()));
+                p->setLinSideBreakValue(*reinterpret_cast<const double(*)[3]>(linSideBreak.data()));
+                p->setLinearSlopeValue(*reinterpret_cast<const double(*)[3]>(linearSlope.data()));
+                p->setDirection(dir);
+                p->validate();
+                return p;
+            }),
+             "logSideSlope"_a = DEFAULT_LOG_SIDE_SLOPE,
+             "logSideOffset"_a = DEFAULT_LOG_SIDE_OFFSET,
+             "linSideSlope"_a = DEFAULT_LIN_SIDE_SLOPE,
+             "linSideOffset"_a = DEFAULT_LIN_SIDE_OFFSET,
+             "linSideBreak"_a = DEFAULT_LIN_SIDE_BREAK,
+             "linearSlope"_a = DEFAULT_LINEAR_SLOPE,
+             "dir"_a = DEFAULT->getDirection())
+
+        .def("getFormatMetadata", 
+             (FormatMetadata & (LogCameraTransform::*)()) &LogCameraTransform::getFormatMetadata,
+             py::return_value_policy::reference_internal)
+        .def("getFormatMetadata", 
+             (const FormatMetadata & (LogCameraTransform::*)() const) 
+             &LogCameraTransform::getFormatMetadata,
+             py::return_value_policy::reference_internal)
+        .def("equals", &LogCameraTransform::equals, "other"_a)
+        .def("getBase", &LogCameraTransform::getBase)
+        .def("setBase", &LogCameraTransform::setBase, "base"_a)
+        .def("getLogSideSlopeValue", [](LogCameraTransformRcPtr self)
+            {
+                std::array<double, 3> values;
+                self->getLogSideSlopeValue(*reinterpret_cast<double(*)[3]>(values.data()));
```

Comment:
Do you need the reinterpret_cast?  Compare this implementation to PyCDLTransform.cpp.

Reply:
The difference between `CDLTransform` and `LogCameraTransform` which requires `reinterpret_cast` here, is in the way `LogCameraTransform` functions expect an array reference instead of a raw array pointer.

`CDLTransform`:
```
//!cpp:function::
virtual void getSlope(double * rgb) const = 0;
//!cpp:function::
virtual void setSlope(const double * rgb) = 0;
```

`LogCameraTransform`:
```
//!cpp:function::
virtual void getLogSideSlopeValue(double(&values)[3]) const noexcept = 0;
//!cpp:function::
virtual void setLogSideSlopeValue(const double(&values)[3]) noexcept = 0;
```

Reply:
Also worth noting that when @hodoulp and I discussed this solution in Slack a while back, we saw it more as a temporary fix until there could be a consistency pass across the OCIO C++ API in regards to array parameters. One proposal was to use a custom vector class.

Reply:
Cool, thanks.  We will plan to do a clean-up pass for RGB values, after it's merged.